src/services/RabbitHoleService.ts

```diff
@@ -1,4 +1,7 @@
-import { apiClient, tryRequest } from '@/api'
+
+import { useApiClient } from '../composables/useApiClient'
+
+const { apiClient, tryRequest } = useApiClient()
```

Comment:
wrong usage, you should not use composables outside of Vue components or Pinia stores.